### PR TITLE
fix travis (flake8)

### PR DESCRIPTION
**Pull request by gurneyalex**

### CODE REVIEW

File: product_customer_code_picking/stock_picking.py [hunk 1]
```diff
@@ -50,9 +50,7 @@ def _get_product_customer_code(
                         ['product_code'], context=context
                     )
                     res[move.id] = (
-                        'product_code' in data
-                        and data['product_code']
-                        or ''
+                        data['product_code'] if 'product_code' in data else ''
```
Comment (pedrobaeza) [inline product_customer_code_picking/stock_picking.py]: Isn't better to put: `data.get('product_code', '')`?

File: stock_picking_compute_delivery_date/model/picking.py [hunk 2]
```diff
@@ -27,9 +27,14 @@
 from openerp.tools.translate import _
 from openerp import pooler
 
+
 _logger = logging.getLogger(__name__)
 
 
+def strptime(string)
```
Comment (lepistone) [inline stock_picking_compute_delivery_date/model/picking.py]: missing `:`

File: stock_obsolete/stock_obsolete.py [hunk 3]
```diff
@@ -41,17 +41,18 @@ def _product_get_multi_location_for_period(self, cr, uid, ids, period,
         # wich determine reference date for out stock computing
         if context.get('ref_date', False):
             date_ref = context['ref_date']
-            past_date_ref = (datetime.strptime(context['ref_date'], '%Y-%m-%d')
-                             + relativedelta(months=-int(period),
-                                             day=1,
-                                             days=-1)).strftime('%Y-%m-%d')
+            past_date_ref = (datetime.strptime(context['ref_date'],
+                                               '%Y-%m-%d') +
```
Comment (pedrobaeza) [inline stock_obsolete/stock_obsolete.py]: Use DEFAULT_SERVER_DATE_FORMAT variable.

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**